pkg/config/config.go

```diff
@@ -97,6 +101,39 @@ func FindConfig() (string, error) {
 		"config.yaml",
 		"config.yml",
 		"config.json",
+		"support.yaml",
+		"support.yml",
+		"support.json",
+	}
+
+	for _, path := range pathsToSearch {
+		for _, filename := range filenamesToSearch {
+			p, err := filepath.Abs(filepath.Join(path, filename))
+			if err != nil {
+				return "", err
+			}
+			if f, err := os.Open(p); err == nil {
+				f.Close()
+				return p, nil
+			}
+		}
+	}
+
+	return "", ErrConfigNotFound
+}
+
+func FindSupportConfig() (string, error) {
```

Comment:
This seems redundant (or is it unused?) given the changes to FindConfig above

Reply:
I'll back out the changs in the FindConfig function as I wanted to be more restrictive here.